### DISCUSSION TITLE
groups for ports

### DISCUSSION THREAD

**Discussion by CodeExplorerX** [pelican-dev/panel#597] [Ideas]
It would be cool to be able to divide the ports into certain groups. For example: Local, TCP, UDP, etc...
And so that on the client side, when issuing ports, it would be possible to select a group to receive the port, and not randomly. 
For example, a user creates a bunch of Minecraft servers, and he only needs one port that goes to the network, and all the others must be local for security, since all connections will go through a proxy server

**Comment by rmartinoscar** [comment]
Hey i believe this is already planned maybe not as deep see https://github.com/pelican-dev/panel/issues/68
